plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -268,7 +268,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				// Used to build serviceLister
 				rbac.NewRule("list", "watch").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
 				rbac.NewRule("get").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
-
+				rbac.NewRule("patch").Groups(legacyGroup).Resources("nodes/status").RuleOrDie(),
```

Comment:
FYI @kubernetes/sig-auth-misc @liggitt @deads2k @jbeda @smarterclayton re allowing kube-proxy to patch nodes/status.